pkg/scheduler/core/generic_scheduler.go

```diff
@@ -293,17 +294,21 @@ func (a TargetClustersList) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 func (a TargetClustersList) Less(i, j int) bool { return a[i].Replicas > a[j].Replicas }
 
 // divideReplicasAggregatedWithResource assigns a total number of replicas to the selected clusters aggregated according to the resource
-func (g *genericScheduler) divideReplicasAggregatedWithResource(clusters []*clusterv1alpha1.Cluster, object *workv1alpha1.ObjectReference) ([]workv1alpha1.TargetCluster, error) {
+func (g *genericScheduler) divideReplicasAggregatedWithResource(clusters []*clusterv1alpha1.Cluster, object *workv1alpha1.ObjectReference,
+	preUsedClustersName ...string) ([]workv1alpha1.TargetCluster, error) {
```

Comment:
Why not using `[]string` instead of `...string`? It seems wired. By the way, it could be a barrier if we want to add more parameters.

Reply:
use "...string" we do not change the logic if first scheduler. If there is no preUsedClusterName , this parameters can be  omitted